test/gtest/ucs/test_log.cc

```diff
@@ -16,6 +16,12 @@ extern "C" {
 class log_test : private ucs::clear_dontcopy_regions, public ucs::test {
 public:
     virtual void init() {
+#ifdef __SANITIZE_ADDRESS__
+        /* BUG: on Ubuntu 22.04 grep returns 1 return code if
+         * LD_PRELOAD=libasan.so even if pattern was found
+         */
+        UCS_TEST_SKIP_R("skipping on asan");
```

Comment:
why is it added now, when testing on ubuntu 22.04 is already merged?

Reply:
Before that patch we passed `-static-libasan` flag, now we changed it to setting `LD_PRELOAD`. I don't know why but this caused wrong `grep` behavior.